src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -724,7 +725,21 @@ const WorkflowView = _.flow(
               disabled: currentSnapRedacted || !!Utils.computeWorkspaceError(ws),
               checked: useCallCache,
               onChange: v => this.setState({ useCallCache: v })
-            }, [' Use call caching'])
+            }, [' Use call caching']),
+            span({ style: { marginLeft: '1rem', marginRight: '0.5rem' } }, [
```

Comment:
If there are multiple margin styles to apply I think the preferred convention is to use `margin: '0 0.5rem 0 1rem'` (which applies the margins clockwise with 0 for top, 0.5 for right, 0 for bottom, and 1 for left)

Reply:
fixed!